s64da_benchmark_toolkit/prepare.py

```diff
@@ -10,7 +10,7 @@
 from concurrent.futures import ThreadPoolExecutor, as_completed
 from jinja2 import Environment, FileSystemLoader
 from pathlib import Path
-from psycopg2 import ProgrammingError
+from psycopg2 import ProgrammingError, errors
```

Comment:
Is that really `errors`? Looks strange somehow...

Reply:
According to this page, errors is correct:
https://www.psycopg.org/docs/errors.html